components/places/src/api/placesapi.rs

```diff
@@ -0,0 +1,256 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use crate::db::db::PlacesDb;
```

Comment:
Given that they're so tightly coupled, I almost feel like the declaration for PlacesDb should be in this file... Hm.